### PR TITLE
Fix bug with GoogleStudentImporter for students with no last name

### PR DESCRIPTION
**Pull request by brendanshean**
## WHAT
Fix a [bug](https://one.newrelic.com/nr1-core/errors/overview/MjYzOTExM3xBUE18QVBQTElDQVRJT058NTQ4ODU2ODc1?account=2639113&duration=1800000&state=7653b600-fb79-8a94-e666-2fc9a08b7fbd) with GoogleStudentClassroomWorker/perform where students have no last name.

## WHY
We'd like these imports to go through

## HOW
Remove FullnameValidator on the user model.

### Screenshots
(If applicable. Also, please censor any sensitive data)

### Notion Card Links
(Please provide links to any relevant Notion card(s) relevant to this PR.)

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  YES
Have you deployed to Staging? | Not yet - deploying now!
Self-Review: Have you done an initial self-review of the code below on Github? | YES
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | N/A

### CODE REVIEW

File: services/QuillLMS/spec/services/generate_fullname_spec.rb [hunk 1]
```diff
@@ -3,51 +3,41 @@
 require 'rails_helper'
```
Comment (dandrabik) [inline services/QuillLMS/spec/services/generate_fullname_spec.rb:3]: Nice thorough specs!

File: services/QuillLMS/app/services/generate_fullname.rb [hunk 2]
```diff
@@ -1,28 +1,23 @@
...
   attr_reader :name
   private :name
 
   private def generate_fullname
-    first_name, last_name = SplitName.new(name).call
-
-    if last_name.nil? && first_name.present?
-      return "#{first_name} #{first_name}"
-    end
+    first_name, last_name = SplitName.run(name)
```
Comment (dandrabik) [inline services/QuillLMS/app/services/generate_fullname.rb:17]: This is a bit code golf-y, so take or leave, but consider:
```ruby
return name unless last_name.nil?

first_name.present? ? "#{first_name} #{first_name}" : DEFAULT_NAME

**Review by dandrabik — APPROVED** [review]
Looks good to me, one small take-or-leave comment. 🚢 

Thanks for fixing this up!

File: services/QuillLMS/app/services/generate_fullname.rb (hunk 2 above)
Comment (brendanshean) [reply services/QuillLMS/app/services/generate_fullname.rb:17]: Thanks.  I think that's more readable.

**Pull request merged**